src/app/app.test.js

```diff
@@ -1,9 +1,10 @@
 import React from 'react'
+import { QueryClient } from 'react-query'
 import { render } from '../test-utils/index.js'
 import App from './app.js'
 
 describe('<App />', () => {
     it('renders without crashing', () => {
-        render(<App />)
+        render(<App />, { queryClient: new QueryClient() })
```

Comment:
The `render` function has a default for the query client, right? Why did you pass a plain query client here?

Reply:
this is the equivalent of how the test was before .. with the default test query client (or the previous implementation with the [QueryClientWrapper](https://github.com/dhis2/data-entry-app/blob/development/src/test-utils/query-client-wrapper.js)) then this test passes but there is a console.error in the logs from trying to get the metadata. I can fix that by passing a CustomDataProvider, or by using `new QueryClient` like the `render` used to default to.

<img width="820" alt="image" src="https://user-images.githubusercontent.com/1014725/180995531-68164f3e-e012-40ff-801f-9548283c556c.png">

<img width="1078" alt="image" src="https://user-images.githubusercontent.com/1014725/180996788-24e365cf-c906-4e75-8542-97aa0441525f.png">